content/integrations/journald.md

```diff
@@ -85,6 +85,16 @@ logs:
       - sshd.service
 ```
 
+##### Collect Container tags
+
+Tags are critical for finding information in highly dynamic containerized environments. Which is why the agent is able to collect container tags in journald logs.
```

Comment:
Tags are critical for finding information in highly dynamic containerized environments, which is why the Agent can collect container tags in journald logs.